mitodl/bootcamp-ecommerce
# Home page base with header - #404

#### What are the relevant tickets?
[Ticket](https://trello.com/c/6ZE6L0pL/77-home-page)
[Ticket](https://trello.com/c/zoks5f0Q/164-home-page-header)

#### What's this PR do?
Adds a `HomePage` class with a base structure. Also modifies URLs to have the CMS serve at the root of the domain.

#### How should this be manually tested?
Create a home page, configure it to be the site main page and verify the live view renders as expected.

#### Screenshots (if appropriate)
Desktop
![screencapture-bc-odl-local-8099-2020-06-01-15_31_15](https://user-images.githubusercontent.com/6387678/83401570-7f4dca00-a41e-11ea-9662-67e6339e1de6.png)

Tablet
![screencapture-bc-odl-local-8099-2020-06-01-15_31_42](https://user-images.githubusercontent.com/6387678/83401578-8379e780-a41e-11ea-94b4-4b058f5cbb03.png)

Mobile
![screencapture-bc-odl-local-8099-2020-06-01-15_32_12](https://user-images.githubusercontent.com/6387678/83401589-883e9b80-a41e-11ea-9a86-46b7e3bd1cf7.png)
